clifford/_layout.py

```diff
@@ -572,9 +560,8 @@ def _gen_complement_func(self, omt):
 
         # Since we're working with basis blades, we can use the table directly.
         # We only care about the pseudo-scalar part of the wedge.
-        omt_ps_part = omt[:, -1, :]
         for n in range(dims):
-            signlist[n] = (-1)**(omt_ps_part[n, dims-1-n] < 0.001)
+            signlist[n] = (-1)**(omt[n, -1, dims-1-n] < 0.001)
```

Comment:
Did you profile this? was it actually faster?

Reply:
After change:
![image](https://user-images.githubusercontent.com/3056964/93327058-b1d6d380-f811-11ea-8468-656ba6292402.png)
Before change:
![image](https://user-images.githubusercontent.com/3056964/93327225-f06c8e00-f811-11ea-8cba-29abfb2f3e5b.png)
Good call! Will revert now :)